### PR TITLE
Basic heuristic planner & Tensorboard handler for DeepEdit

### PR DESCRIPTION
**Pull request by diazandr3s**
Signed-off-by: Andres diazandr3s@gmail.com

- Create a basic heuristic training planner considering spacing, image size and GPU memory available
- Add Tensorboard handler for simulated clicks analysis - DeepEdit
- Change execution order in init for interfaces/app.py

### CODE REVIEW

**Comment by diazandr3s** [comment]
@pritesh-mehta please check these changes work on your computer

**Comment by SachidanandAlle** [comment]
Don't remove the license header.. also we need it on new files

File: sample-apps/deepedit_spleen/main.py [hunk 1]
```diff
@@ -1,14 +1,3 @@
-# Copyright 2020 - 2021 MONAI Consortium
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (SachidanandAlle) [inline sample-apps/deepedit_spleen/main.py:5]: you can should be this back

File: sample-apps/deepedit_spleen/lib/train.py [hunk 2]
```diff
@@ -137,6 +139,7 @@ def val_iteration_update(self):
         )
```
Comment (SachidanandAlle) [inline sample-apps/deepedit_spleen/lib/train.py:139]: https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/utils/train/basic_train.py#L206

Override this method to pass addition event names which are expecting to get fired.. in this case.. you are using events defined in IterationEvents.xyz

**Comment by SachidanandAlle** [comment]
please fix the build..

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**